docker/local/README.md

```diff
@@ -0,0 +1,39 @@
+## Turbinia local stack using Docker
+Turbinia can be run locally without any Cloud components using Docker. It will use Redis, Celery and local disk to store data and perform message broker functionality.
+
+### Caveats
+rawdisk: As Turbinia uses the loop device to mount different types of evidence (eg raw disks) the host operating system should support the loop device. Linux is currently the only OS that supports the processing of raw disks. It is possible to process different evidence types, eg compresseddirectory, using the local stack on OSX or Windows.
+
+googleclouddisk: Turbinia as local stack can currently not process Google Cloud disks.
+
+### Running
+
+#### Step 1
+Checkout the Turbinia source code. If you intend to start developing please fork the repository on github first and check out your own forked instance.
+```
+$ git clone https://github.com/google/turbinia.git
+$ cd turbinia
+```
+#### Step 2
+Copy the configuration template for the local stack to the ```./conf`` folder (create the folder if it does not exist).
+```
+$ cp docker/local/turbinia-local-template.conf ./conf/turbinia.conf
+```
+#### Step 3
+Let's bring up the local Turbinia stack
+```
+$ docker-compose -f ./docker/local/docker-compose.yml up
+```
+Redis, a Turbinia server and worker should now be running on your local system and a local persistent 'evidence' folder will have been created containing the Turbinia log file and processing output.
+#### Step 4
+Let's process evidence to test your setup, eg a Chrome Browser history file.
+```
+$ curl https://raw.githubusercontent.com/obsidianforensics/hindsight/master/tests/fixtures/profiles/60/History > History
+$ tar -vzcf ./evidence/history.tgz History
+$ docker exec -ti turbinia-server turbiniactl compresseddirectory -l /evidence/history.tgz
```

Comment:
What would it take for us to be able to run the `turbiniactl` commands outside of the container?

Also, maybe we can add a little more info on exactly what this is doing here?  Maybe something like "This command runs the turbinia client, turbiniactl, within the turbinia server docker container and generates a processing request"

Reply:
You would need to install turbinia (including all dependencies....) on your local host (preferably in a virtual env). I would not recommend that as it will create only more problems. And you already have a known good turbiniactl in the docker container. By writing it down like this we also prevent people from filing issues and us trying to debug their local setup.
Added a clarification to what it does.

Reply:
Ok, we can leave it for now then, and once we get the client refactored with fewer dependencies it will be easier to deploy this independently.  FWIW, we do have instructions for doing this with a virtualenv in the cloud deployment.